assets/src/clear-specification-and-implementation/extract.pl

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
```

Comment:
It was the easiest way to write this, IMO, and it's pretty reliably available everywhere. Other languages would certainly work, though.

Reply:
> It was the easiest way to write this, IMO

I can't comment on that, since I'm not sure how to read this code.

I think it's best to minimize the number of programming languages used in a particular repository/project. For Dafny I think it would be better not to have any Perl in there. We already have Bash and Python as scripting languages in the main Dafny repository, so I'd recommend biasing for those.

Reply:
I replaced it with Bash.